lms/static/js/instructor_dashboard/student_admin.js

```diff
@@ -553,7 +560,10 @@
                     success: this.clear_errors_then(function() {
                         return alert(fullSuccessMessage);  // eslint-disable-line no-alert
                     }),
-                    error: statusAjaxError(function() {
+                    error: statusAjaxError(function(response) {
+                        if (response.responseText === 'Specified module does not support rescoring.') {
+                            return that.$request_err_grade.text(rescoreUnsupportedMessage);
+                        }
```

Comment:
@sanfordstudent This seems like it hides the error relative to the success return alert() -- what's the surfacing of this to the user?